evalml/data_checks/highly_null_data_check.py

```diff
@@ -84,7 +110,7 @@ def validate(self, X, y=None):
             percent_null_rows >= self.pct_null_row_threshold
         ]
         if len(highly_null_rows) > 0:
-            warning_msg = f"{len(highly_null_rows)} out of {len(X)} rows are more than {self.pct_null_row_threshold*100}% null"
+            warning_msg = f"{len(highly_null_rows)} out of {len(X)} rows are {self.pct_null_row_threshold*100}% or more null"
```

Comment:
Same comment as in the sparsity data check: Just wondering we were modifying this.

Reply:
Because the threshold check is `percent_null_rows >= self.pct_null_row_threshold`, so the phrasing `or more` seemed more appropriate than `more than`